packages/design-system/src/icons/index.js

```diff
@@ -146,3 +146,6 @@ export { ReactComponent as TurningLine } from './turning_line.svg';
 export { ReactComponent as Union } from './union.svg';
 export { ReactComponent as Video } from './video.svg';
 export { ReactComponent as Launch } from './launch.svg';
+
+// This icon is also needed as a raw SVG for CSS
+export { default as ArrowCircleSVG } from './arrow_circle.svg';
```

Comment:
This is not optimal. As mentioned in the PR description, I think it makes more sense to put these in a new object as `Icons.svg.arrowCircle` or something similar (and make sure that the Storybook story for icons ignores the `svg` object).

Thoughts?

Reply:
Some non-standard icons we've also added directly to the folder it's used in, I think. If it's used only in one place and is non-standard, might also be an option to just import it from the same folder to avoid design system icons issues.

Alternatively sgtm to add a new folder for svg-s and ignore those in the Storybook.

Reply:
I decided to go for a slightly different and cleaner approach. The reason why I kept it here (in the design system) is that it's the exact same icon that we use elsewhere as a regular component, so didn't want to make a duplicate.